docs/docs/user-setup.md

```diff
@@ -77,6 +78,12 @@ bash -c "$(curl -fsSL https://zmkfirmware.dev/setup.sh)"
 bash -c "$(wget https://zmkfirmware.dev/setup.sh -O -)"
 ```
 
+</TabItem>
+<TabItem value="PowerShell">
+
+```
+iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/CrossR/zmk/CrossR/ShellScriptTidyUp/docs/static/setup.ps1'))"
```

Comment:
The URL here should be https://zmkfirmware.dev/setup.ps1 now, the old location was just for testing before it was published to the website.

Reply:
Thank you!